docs/core/transactions.md

```diff
@@ -0,0 +1,84 @@
+# Transactions
+
+## Prerequisites
+
+* [Anatomy of an SDK Application](./app-anatomy.md)
+
+## Synopsis
+
+This document describes how various components are defined to enable transactions. It also describes how transactions are generated.
+
+1. [Transactions](#transactions)
+2. [Transaction Definition](#transaction-definition)
+3. [CLI and REST Interfaces](#cli-and-rest-interfaces)
+4. [Messages](#messages)
+5. [Transaction Generation](#transaction-generation)
+6. [Handlers](#handlers)
+
+
+## Transactions
+
+[**Transactions**](https://github.com/cosmos/cosmos-sdk/blob/master/types/tx_msg.go#L36-L43)  are objects that wrap messages and transaction data, created to enable a user to interact with an application. Specifically, they are comprised of metadata held in [contexts](./contexts) and [messages](../modules.md#messages) that trigger state changes within a module through the module's Handler.
+
+When users want to interact with an application and make state changes (e.g. sending coins), they create transactions. Each of a transaction's messages must be signed using the private key associated with the appropriate account(s), and then the transaction is broadcasted to the network. A transaction must then be included in a block, validated, and approved by the network through the consensus process. To read more about the lifecycle of a transaction, click [here](../basics/tx-lifecycle.md).
+
+## Transaction Definition
+
+Transactions messages are defined by module developers. The transaction objects themselves are SDK types that implement the [`Tx`](https://github.com/cosmos/cosmos-sdk/blob/master/types/tx_msg.go#L34-L41) interface and include an encoder and decoder:
```

Comment:
The first sentence should be under `GetMsgs`. The section should not start with messages if it's about transactions, cause that's confusing